src/Core/Framework/Plugin/Command/PluginListCommand.php

```diff
@@ -36,22 +36,30 @@ protected function configure(): void
     {
         $this
             ->setDescription('Show a list of available plugins.')
-            ->addOption('filter', 'f', InputOption::VALUE_REQUIRED, 'Filter the plugin list to a given term');
+            ->addOption('filter', 'f', InputOption::VALUE_REQUIRED, 'Filter the plugin list to a given term')
+            ->addOption('format', null, InputOption::VALUE_REQUIRED, 'Choose table or JSON format to output to', 'table');
     }
 
     /**
      * {@inheritdoc}
      */
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        $format = strtolower($input->getOption('format'));
```

Comment:
When its xml you output nothing, no errors

Reply:
I now throw an error